### PR TITLE
1 4 2

### PR DESCRIPTION
**Pull request by pwgit-create**

### CODE REVIEW

**Review by lilstiffy — APPROVED** [review]
PR looks good, some indentation related stuff that can be cleaned up ony :D

File: cg/CodeGenerator/CodeGenerator/src/main/java/pn/cg/util/StringUtil.java [hunk 1]
```diff
@@ -159,5 +162,151 @@ public static String RemoveEveryThingAfterEndChar(String input){
...
+    
+
+        }
+            else if(!isStartDelimiter){
+
+            delimiter = CommonStringConstants.JAVA_CODE_GENERATION_END_DELMITER_STRING;
+              returnValue = RemoveAllExceptTheLastOccurrenceofAWord(returnValue, delimiter);
+
+        }
+        }
```
Comment (lilstiffy) [inline cg/CodeGenerator/CodeGenerator/src/main/java/pn/cg/util/StringUtil.java:225]: here for instance :D

File: cg/CodeGenerator/CodeGenerator/src/main/java/pn/cg/util/StringUtil.java [hunk 2]
```diff
@@ -159,5 +162,151 @@ public static String RemoveEveryThingAfterEndChar(String input){
...
+    
+
+    catch (Exception e) {
+        log.debug("Error on delimiter String removal process");
+    }
+
+        return returnValue;
```
Comment (lilstiffy) [inline cg/CodeGenerator/CodeGenerator/src/main/java/pn/cg/util/StringUtil.java:232]: indent

File: cg/CodeGenerator/CodeGenerator/src/main/java/pn/cg/util/StringUtil.java (hunk 1 above)
Comment (pwgit-create) [reply cg/CodeGenerator/CodeGenerator/src/main/java/pn/cg/util/StringUtil.java:225]: Thanks :)

File: cg/CodeGenerator/CodeGenerator/src/main/java/pn/cg/util/StringUtil.java (hunk 2 above)
Comment (pwgit-create) [reply cg/CodeGenerator/CodeGenerator/src/main/java/pn/cg/util/StringUtil.java:232]: Ty :smiley:

**Pull request merged**